src/core/file.cc

```diff
@@ -1346,6 +1347,32 @@ file_impl::dup() {
     throw std::runtime_error("this file type cannot be duplicated");
 }
 
+#ifdef SEASTAR_COROUTINES_ENABLED
+coroutine::experimental::generator<directory_entry, dir_entry_buffer> make_list_directory_fallback_generator(coroutine::experimental::buffer_size_t buffer_size, file_impl& me) {
```

Comment:
might want to mark this function `static` or move it into an anonymous namespace.

Reply:
Good point. I had this habit to mark .cc-local methods `static` from my C past, but then abandoned it for some reason